stable/agent/values.yaml

```diff
@@ -22,8 +22,7 @@ enableHostDocker: true
 
 # Enable security context configuration
 securityContext:
```

Comment:
if nothing set then there should be `securityContext: {}`

Reply:
I wondered that but then saw the `extraEnv` doesn't have a default like that

Reply:
as `extraEnv` is not part of the template spec, but `securityContext` is.

Reply:
Oh okay thanks for clearing that up

Reply:
sorry my bad we have `{{- if .Values.securityContext.enabled }}` so it has to be `securityContext:`.
sorry for confusion.

Reply:
But this PR changes that to use the `with` syntax so the `securityContext.enabled` property is no longer used. I think I've fixed it with the last push